src/languages/stan.js

```diff
@@ -27,21 +28,41 @@ export default function(hljs) {
     'continue',
     'return'
   ];
+
   const SPECIAL_FUNCTIONS = [
     'print',
     'reject',
-    'increment_log_prob|10',
-    'integrate_ode|10',
-    'integrate_ode_rk45|10',
-    'integrate_ode_bdf|10',
-    'algebra_solver'
+    'hmm_marginal',
+    'hmm_latent_rng',
+    'hmm_hidden_state_prob',
+    'algebra_solver',
+    'algebra_solver_newton',
+    'map_rect',
+    'reduce_sum',
+    'reduce_sum_static',
+    'ode_rk45', 
+    'ode_rk45_tol', 
+    'ode_ckrk', 
+    'ode_ckrk_tol', 
+    'ode_adams',
+    'ode_adams_tol', 
+    'ode_bdf', 
+    'ode_bdf_tol', 
+    'ode_adjoint_tol_ctl',
+    'integrate_1d',
+    'integrate_ode_rk45', 
+    'integrate_ode', 
+    'integrate_ode_adams',
+    'integrate_ode_bdf'
   ];
   const VAR_TYPES = [
+    `array`,
+    'complex',
     'int',
     'real',
     'vector',
     'ordered',
-    'positive_ordered',
+    'positive_ordered|10',
```

Comment:
Justification for 10 here?

Reply:
are there other languages which use `positive_ordered` types? I thought this was one of the few